packages/docs/components/page-configs/va-checkbox/api-options.ts

```diff
@@ -0,0 +1,40 @@
+import { ManualApiOptions } from 'vuestic-ui/src/services/api-docs/ManualApiOptions'
+
+const options: ManualApiOptions = {
+  props: {
+    value: {
+      local: true,
+    },
+    arrayValue: {
+      local: true,
+    },
+    label: {
+      local: true,
+    },
+    leftLabel: {
+      local: true,
+    },
+    trueValue: {
```

Comment:
Any reason to keep those local? They're part of `SelectableMixin`.
For full list of related props - check said mixin.